site/src/pages/TemplateSettingsPage/TemplateSettingsForm.tsx

```diff
@@ -1,11 +1,17 @@
+import data from "@emoji-mart/data/sets/14/twitter.json"
+import Picker from "@emoji-mart/react"
+import Button from "@material-ui/core/Button"
 import InputAdornment from "@material-ui/core/InputAdornment"
+import Popover from "@material-ui/core/Popover"
 import { makeStyles } from "@material-ui/core/styles"
 import TextField from "@material-ui/core/TextField"
+import ExpandMore from "@material-ui/icons/ExpandMore"
```

Comment:
This sounds like the same thing as our `OpenDropdown`, maybe we should just use one of them everywhere?

Reply:
I agree! Replacing it.